lib/dialects/mysql/index.js

```diff
@@ -162,16 +162,16 @@ class Client_MySQL extends Client {
   }
 
   async cancelQuery(connectionToKill) {
-    const conn = await this.acquireConnection();
+    const conn = await this.acquireRawConnection();
     try {
-      return await this.query(conn, {
+      return await this._query(conn, {
         method: 'raw',
```

Comment:
probably extra field (in pg too)

Reply:
What do you mean by "extra"?

Reply:
Oh, I see `method: 'raw'` field is not used in driver-level `_query`. It was just field used by knex-level `query` layer. I will remove the field. The same for Postres dialect.

Reply:
Removed in 0787821